.github/workflows/publish-test-results.yml

```diff
@@ -58,7 +58,8 @@ jobs:
         
       - name: Update test reporter
         run: |
-          sed 's#module.exports = defineConfig({#module.exports = defineConfig({\n  reporter: '\"'mochawesome'\"',\n  reporterOptions: {\n    reportDir: '\"'cypress/reports/mocha'\"',\n    quiet: true,\n    overwrite: false,\n    html: false,\n    json: true\n  },#g' ${{ env.config_file }} > temp
+          sed 's#module.exports = defineConfig({#module.exports = defineConfig({\n  reporter: '\"'mochawesome'\"',\n  reporterOptions: {\n    reportDir: '\"'cypress/reports/mocha'\"',\n    quiet: true,\n    overwrite: false,\n    html: false,\n    json: true\n  },#g' ${{ env.config_file }} > temp_js
+          sed 's#export default defineConfig({#export default defineConfig({\n  reporter: '\"'mochawesome'\"',\n  reporterOptions: {\n    reportDir: '\"'cypress/reports/mocha'\"',\n    quiet: true,\n    overwrite: false,\n    html: false,\n    json: true\n  },#g' temp_js > temp
```

Comment:
We do 2 passes of `sed` only 1 is expected to match, this should be backward compatible since if no match is found the original content is written. 
We now support both cypress js/ts structures as specified in the documentation: https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file